docs: dracut-crypt-ssh keys name and format fix (#743)
zbm-dev/zfsbootmenu: docs/general/remote-access.rst

   format.

To create dedicated host keys in the proper format, decide on a location, for example ``/etc/dropbear``, and create the
new keys:

.. tabs::

  .. group-tab:: Dracut

>       The dracut module expects to install RSA and ECDSA keys, so at minimum those keys should be created
>       We're making keys in PEM format because the dracut module insists on converting keys each time and doesn't accept keys in dropbear format::

Reviewer: Avoid first-person voice and provide a bit more exposition.
```suggestion
      The dracut module expects to install RSA and ECDSA keys, so at least these keys should be created. The ``dracut-crypt-ssh`` module forces the use of the ``dropbearconvert`` utility during creation of the initramfs image to convert OpenSSH keys into the format used by ``dropbear``. Care must be taken to ensure that OpenSSH keys are created in the PEM format expected by ``dropbearconvert``::
```

        
        mkdir -p /etc/dropbear
        for keytype in rsa ecdsa ed25519; do

>           ssh-keygen -g -N "" -m PEM -t "${keytype}" -f "/etc/dropbear/ssh_dracut_host_${keytype}_key"

Reviewer: Leave the output files named as the later directions require, rather than changing those as well.
```suggestion
          ssh-keygen -g -N "" -m PEM -t "${keytype}" -f "/etc/dropbear/ssh_host_${keytype}_key"
```

        done

  .. group-tab:: mkinitcpio

>     The mkinitcpio module supports RSA, ECDSA, and ED25519 keys::

Reviewer: ```suggestion
For mkinitcpio, generate keys for all supported types::
```

        
        mkdir -p /etc/dropbear
        for keytype in rsa ecdsa ed25519; do
          dropbearkey -t "${keytype}" -f "/etc/dropbear/dropbear_${keytype}_host_key"

>         done

Reviewer: ```suggestion
        done
        
        Not all versions of ``dropbear`` support ED25519 keys, so it is fine if creation of the ED25519 key fails.
```

   format.

To create dedicated host keys in the proper format, decide on a location, for example ``/etc/dropbear``, and create the
new keys:

.. tabs::

  .. group-tab:: Dracut

>       The dracut module expects to install RSA and ECDSA keys, so at least these keys should be created. The ``dracut-crypt-ssh`` module forces the use of the ``dropbearconvert`` utility during creation of the initramfs image to convert OpenSSH keys into the format used by ``dropbear``. Care must be taken to ensure that OpenSSH keys are created in the PEM format expected by ``dropbearconvert``::

Reviewer: Please wrap this to a width consistent with the rest of the document.

      # Enable dropbear ssh server and pull in network configuration args
      add_dracutmodules+=" crypt-ssh "
      install_optional_items+=" /etc/cmdline.d/dracut-network.conf "
      # Point to host keys made earlier
      dropbear_rsa_key=/etc/dropbear/ssh_dracut_host_rsa_key

>       dropbear_ecdsa_key=/etc/dropbear/ssh_dracut_host_ecdsa_key

Reviewer: Drop changes in light of edits above.

Author: Perhaps "Copy system keys instead of separately generated ones to have consistent access" would be better?
I still think the naming is a bit confusing in current state

Reviewer: "Use pre-generated keys for consistent access"
